### ISSUE TITLE
Broken Bibliography Citation Links

### ISSUE THREAD

**Issue by ChaosMageX** [Nu-SCPTheme/Black-Highlighter#186]
**Describe the issue:**  
Bibliography citation links appear on separate lines. It seems that the `display: inline-block` property of `a.bibcite` elements is normalized but never re-implemented.

**Screenshot showcasing the issue:**  
![image](https://github.com/Nu-SCPTheme/Black-Highlighter/assets/8120512/49790240-e3ac-481c-8a8f-9799060348d5)

**Link to the page(s) having this issue:**  
http://scptestwiki.wikidot.com/cmxtest2

**Device information:** (fill in as appropriate)  
 - Device: Custom Built PC (Do I need to provide full specs?)
 - OS: Windows 10 (64-bit)
 - Browser: Chrome
 - Version: 112.0.5615.138 (Official Build) (64-bit)

**Additional context:**  
This can be tested on any page using the BHL Theme. I literally just copy/pasted the example code from the Wikidot documentation onto one of my test pages:
http://www.wikidot.com/doc-wiki-syntax:bibliography

The [[bibliography]] feature might not be used much on the SCP Wiki, if at all, but it's still worth fixing this issue, just in case somebody does eventually use it, maybe in a Tale or a GOI Format Page.

**Comment by Woedenaz** [comment]
This appears to be caused by the `text-indent: 2ch;` on `.bibitems .bibitem`.

Since the bibliography looks perfectly find on the base BHL theme, this is probably something I'll need to correct on the UIU theme itself.

**Comment by ChaosMageX** [comment]
No, I'm talking about the **Citation Links**, which end up on separate lines, even on the base BHL Theme:

![image](https://github.com/Nu-SCPTheme/Black-Highlighter/assets/8120512/28c06b74-c40a-4fb2-8d40-6ad455e5a66e)

Right now, it can only be fixed by disabling the `display: inherit` property under the `a.bibcite` selector in **normalize.css**, which causes the property to revert back to `display: inline-block`, as defined by Wikidot's Common Theme:

![image](https://github.com/Nu-SCPTheme/Black-Highlighter/assets/8120512/8008ab58-7b41-4628-b652-3caeb318823b)

**Comment by Woedenaz** [comment]
Oh! I see. I didn't even notice that. Thank you, I'll give that a looksee.

**Comment by SYwaves** [comment]
This has been fixed.
